src/Process.php

```diff
@@ -82,6 +84,10 @@ public function start(LoopInterface $loop, $interval = 0.1)
             throw new \RuntimeException('Process is already running');
         }
 
+        if (defined('PHP_WINDOWS_VERSION_BUILD') && $this->windowsWorkaround) {
```

Comment:
We can safely remove the `defined()` check here and rely on the other flag. This should also help with testing?